plugins/modules/system/sefcontext.py

```diff
@@ -246,6 +274,79 @@ def semanage_fcontext_delete(module, result, target, ftype, do_reload, sestore='
     module.exit_json(changed=changed, **result)
 
 
+def semanage_fcontext_equivalence(sefcontext, target):
+    ''' Test if equivalence exists for target in policy. Return None if it does not exist. '''
+    try:
+        # this is for the shipped equivs.
+        return sefcontext.equiv_dist[target]
+    except KeyError:
+        try:
+            # these are the local added items.
+            return sefcontext.equiv[target]
+        except KeyError:
+            return None
```

Comment:
This can be simplified as:
```suggestion
    return sefcontext.equiv_dist.get(target, sefcontext.equiv.get(target))
```